### ISSUE TITLE
Problem in the outer product of two 1D chebfun objects

### ISSUE THREAD

**Issue by bhashemi** [chebfun/chebfun#1989]
There are currently two issues if one wants to compute a `chebfun2` as the outer product of two 1D chebfuns (or quasimatrices). The first one is that the code that does this is in the chebfun2 folder and NOT in the chebfun folder which means one has to say 
```
F = chebfun2.outerProduct(g, h');
```
and not just 
```
F = outerProduct(g, h');
```
The second problem is evident above as one has to transpose the second input.

**Comment by ajt60gaibb** [comment]
Can one do `g*h'`?

**Comment by bhashemi** [comment]
That's right. The problem is more of a syntax type: the transposition in `g*h'` cannot be extended to 3D without confusion. The `outerProduct` is the right syntax in my view.
